lib/client.js

```diff
@@ -221,14 +221,14 @@ function processResponse(path, result) {
 
 function isOriginValid(origin) {
   var WHITELISTED_ORIGINS = [
-    /^http(s)?:\/\/127.0.0.1(:\d+)?$/,
-    /^http(s)?:\/\/localhost(:\d+)?$/,
-    /^https:\/\/\w+\.zendesk\.com$/,
```

Comment:
just wondering what's wrong with `\w` cause it means letter, number, underscore. sounds like a url to me. `.` will also match spaces etc.

Reply:
It doesn't work with dashes, and who knows what else...

Reply:
who has dashes in their url!?!?!?! :trollface: